Tasks/AzureFileCopyV4/AzureUtilityARM.ps1

```diff
@@ -55,7 +55,7 @@ function Create-AzureContainer
         {
             $container = New-AzureStorageContainer -Name $containerName -Context $storageContext -ErrorAction Stop
         } else {
-            $container = New-AzureStorageContainer -Name $containerName -Context $storageContext -Permission Container -ErrorAction Stop
+            $container = New-AzureStorageContainer -Name $containerName -Context $storageContext -Permission Off -ErrorAction Stop
```

Comment:
what does '-permission off' mean ?

Reply:
@arjgupta Permission Off means that access is restricted to only the storage account owner. When we set this permissiom, the container is created with private mode. Earlier it was getting created with container access level.
If we run AzureFileCopyV3 task, in that also container gets created with private mode only.

![image](https://user-images.githubusercontent.com/45754171/97546101-f600ea80-19f1-11eb-8e18-f95c72b4bf45.png)